packaging/python_wheel/setup.py.in

```diff
@@ -14,8 +11,12 @@ def has_ext_modules(self):
 
 setup(
     name='brain',
-    version=VERSION,
-    description='brain',
+    version='${Brion_VERSION}',
```

Comment:
The build process doesn't currently create a version.py, which would probably make sense.